src/components/userAccount/UserAccount.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Image } from "components/Image";
+import { FlexWrapper } from "components/wrappers/FlexWrapper";
+import { Box } from "components/wrappers/Box";
+import { Typography } from "components/typography/Typography";
+
+type UserAccountType = {
```

Comment:
for prop typing, prefer using the interface. props are objects and interfaces are designed for typing objects